README.md

```diff
@@ -5,7 +5,7 @@
 Airsonic
 ========
 [![Build Status](https://travis-ci.org/airsonic/airsonic.svg?branch=master)](https://travis-ci.org/airsonic/airsonic)
-[![Coverity scan](https://scan.coverity.com/projects/17971/badge.svg)](https://scan.coverity.com/projects/airsonic)
+[![Coverity scan](https://scan.coverity.com/projects/18356/badge.svg)](https://scan.coverity.com/projects/airsonic-airsonic)
```

Comment:
why the change in url?

Reply:
Because the previous one is pointing to my fork, the new one is the coverity instead for this repository :)

Reply:
Hmm, both seem to point to https://github.com/airsonic/airsonic.git but the one you're changing to doesn't list Javascript as a secondary language so only checks about 40k lines of code instead of nearly 80k lines of code.  The previous one also has more history.

Reply:
The badge on the [project's webpage]( https://scan.coverity.com/projects/airsonic?tab=overview ) is at https://scan.coverity.com/projects/17971/badge.svg?flat=1

About history and coverage, those are outdated: the project wasn't scanned since April 2019, where I launched it manually, while the previous one is still running on my fork I think.